src/classes/generator.ts

```diff
@@ -1,27 +1,39 @@
-import {Feature, Prototyper} from '/home/willian/Projects/tcc/ui-core'
+import {Feature, Prototyper} from '/Users/vcpablo/Code/concordia-ui/ui-core/src'
 
 import ElementFactory from './element-factory'
 import {HtmlElement} from './html-elements'
 
+import * as fs from 'fs'
+
 export default class Generator implements Prototyper {
-  public generate(features: Feature[]): Promise<string[]> {
+  public write(feature: Feature, elements: HtmlElement[]) {
```

Comment:
- [ ] Please use `async` writes/reads. It's easier to use a promisified version of `writeFile`.